src/main/java/redis/clients/jedis/Observability.java

```diff
@@ -0,0 +1,259 @@
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/dials"),
+                    "The number of dials",
+                    MDials,
+                    countAggregation,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/dial_latency"),
+                    "The number of milliseconds spent dialling",
+                    MDialLatencyMilliseconds,
+                    defaultMillisecondsDistribution,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/bytes_read_count"),
+                    "The number of bytes read back from the server",
+                    MBytesRead,
+                    countAggregation,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/bytes_read_distribution"),
+                    "The number of bytes read back from the server",
+                    MBytesRead,
+                    defaultBytesDistribution,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/bytes_written_cumulative"),
+                    "The number of bytes written to the server",
+                    MBytesWritten,
+                    countAggregation,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/bytes_written_distribution"),
+                    "The number of bytes written to the server",
+                    MBytesWritten,
+                    defaultBytesDistribution,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/roundtrip_latency"),
+                    "The distribution of milliseconds",
+                    MRoundtripLatencyMilliseconds,
+                    defaultMillisecondsDistribution,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/write_errors"),
+                    "The number of errors encountered while writing to the server",
+                    MWriteErrors,
+                    countAggregation,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/writes"),
+                    "The number of writes",
+                    MWrites,
+                    countAggregation,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/reads"),
+                    "The number of reads",
+                    MReads,
+                    countAggregation,
+                    noKeys),
+
+            View.create(
+                    Name.create("redis/client/read_errors"),
+                    "The number of read errors",
+                    MReadErrors,
+                    countAggregation,
+                    noKeys)};
+
+
+        ViewManager vmgr = Stats.getViewManager();
```

Comment:
nit: `ViewManager` could be `private static final`

Reply:
Thanks @songy23, I'll fix that.

Reply:
Oh wait, it is only used in that function and in about only 3 lines, so perhaps not much of a worry?

Reply:
Yes in this demo code it's fine to leave as-is. For common use cases, we usually need to keep a reference to ViewManager (as a static variable). For example the [quick start](https://github.com/census-instrumentation/opencensus-java/blob/master/examples/src/main/java/io/opencensus/examples/helloworld/QuickStart.java#L51).

Reply:
This isn't demo code though :) This is the actual instrument client code that will be used by everyone. I thought about making a static client yesterday but this function only gets called once and that variable is used only down here below so didn't necessitate that.

Reply:
SGTM